_api-reference/index-apis/get-settings.md

```diff
@@ -40,7 +40,7 @@ Parameter | Data type | Description
 allow_no_indices | Boolean | Whether to ignore wildcards that don’t match any indexes. Default is `true`.
 expand_wildcards | String | Expands wildcard expressions to different indexes. Combine multiple values with commas. Available values are `all` (match all indexes), `open` (match open indexes), `closed` (match closed indexes), `hidden` (match hidden indexes), and `none` (do not accept wildcard expressions), which must be used with `open`, `closed`, or both. Default is `open`.
 flat_settings | Boolean | Whether to return settings in the flat form, which can improve readability, especially for heavily nested settings. For example, the flat form of “index”: { “creation_date”: “123456789” } is “index.creation_date”: “123456789”.
-include_defaults | String | Whether to include default settings, including settings used within OpenSearch plugins, in the response. Default is false.
+include_defaults | Boolean | Whether to include default settings, including settings used within OpenSearch plugins, in the response. Default is false.
```

Comment:
```suggestion
include_defaults | Boolean | Whether to include default settings, including settings used within OpenSearch plugins, in the response. Default is `false`.
```

Reply:
Noticed @Naarcha-AWS committed the change, thanks.

I opened https://github.com/opensearch-project/documentation-website/pull/7660 for the other instances of this.